### ISSUE TITLE
training with custom loss function

### ISSUE THREAD

**Issue by fhamborg** [explosion/spacy-pytorch-transformers#73]
Hi! 

Is it possible to define a custom loss function when training spacy's transformers? 

And also, I was wondering, why in `train_textcat.py` the evaluation function ignores documents where the predicted label != the goldlabel, cf. https://github.com/explosion/spacy-pytorch-transformers/blob/ab132b674c5a91510eb8cc472cdbdf5877d24145/examples/train_textcat.py#L260 Doesn't this skew the evaluation, since we would also need to evaluate true positives, false positives, and so on on the non-positive label?

Thanks in advance for your answer! :-)

Cheers,
Felix

**Comment by pbakhtin** [comment]
Also want to know this for #77
